### PR TITLE
sql: Add placement columns for `show table regions`

### PR DESCRIPTION
**Pull request by lcwangchao**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/10735
- Other reference link(s): https://github.com/pingcap/tidb/pull/35202

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-table-regions.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ Executing `SHOW TABLE REGIONS` returns the following columns:
 * `READ_BYTES`: The estimated amount of data read from the Region within one heartbeat cycle. The unit is byte.
 * `APPROXIMATE_SIZE(MB)`: The estimated amount of data in the Region. The unit is megabytes (MB).
 * `APPROXIMATE_KEYS`: The estimated number of Keys in the Region.
+* `SCHEDULING_CONSTRAINTS`：The [placement rule](/placement-rules-in-sql.md) of the region which belongs to a table or partition with placement setting.
```
Comment (xhebox) [inline sql-statements/sql-statement-show-table-regions.md:53]: We should call it `placement policy`. Only use `placement rules` if we are talking the feature name.

File: sql-statements/sql-statement-show-table-regions.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ Executing `SHOW TABLE REGIONS` returns the following columns:
 * `READ_BYTES`: The estimated amount of data read from the Region within one heartbeat cycle. The unit is byte.
 * `APPROXIMATE_SIZE(MB)`: The estimated amount of data in the Region. The unit is megabytes (MB).
 * `APPROXIMATE_KEYS`: The estimated number of Keys in the Region.
+* `SCHEDULING_CONSTRAINTS`：The [placement policy settings](/placement-rules-in-sql.md) of the region which belongs to a table or partition with placement setting.
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-table-regions.md:53]:
```suggestion
* `SCHEDULING_CONSTRAINTS`: The [placement policy settings](/placement-rules-in-sql.md) of the region which belongs to a table or partition with placement setting.
```

File: sql-statements/sql-statement-show-table-regions.md [hunk 3]
```diff
@@ -50,6 +50,8 @@ Executing `SHOW TABLE REGIONS` returns the following columns:
...
+* `SCHEDULING_STATE`：The scheduling state for the region which has a placement policy.
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-table-regions.md:54]:
```suggestion
* `SCHEDULING_STATE`: The scheduling state for the region which has a placement policy.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**